src/utils/URLParamsParser.ts

```diff
@@ -0,0 +1,14 @@
+export function paramsParser(queryParams: Record<string, string>) {
+	let parsedParams = '';
```

Comment:
1. Use reduce then you do not have to append an extra '&' and rmv it
2. This function should accept null | undefined and return an empty string for invalid params so that you don't have to write conditional operators on each endpoint where you need a query string. Like this: https://github.com/parseablehq/console/pull/289/files#diff-dc3796761041251e6580415307413937f036d7143b52eb0a254085147ac758bcR8
3. The '?' itself should be emitted from this function.
4. If you make all these changes, the fn name is inappropriate. It should mention what it receives and what it returns.